src/components/search-box.vue

```diff
@@ -7,7 +7,10 @@
       class="desktop:x-pl-24"
     />
 
-    <ClearSearchInput v-if="isDesktopOrGreater && $x.query.searchBox" class="x-input-group-button">
+    <ClearSearchInput
+      v-if="isDesktopOrGreater && $x.query.searchBox"
+      class="x-button x-input-group-button x-input-group-button-rectangle"
```

Comment:
Hi! `x-button` class is contained in the ClearSearchInput component, so we can remove it from here:
```suggestion
      class="x-input-group-button x-input-group-button-rectangle"
```